rm_vk/rm_vk/Extension/UITextField+Extension.swift

```diff
@@ -0,0 +1,13 @@
+// UITextField+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit.UITextField
```

Comment:
```suggestion
import UIKit
```
мне кажется не обязательно через точку прописывать